swc/defs.bzl

```diff
@@ -146,9 +146,14 @@ def swc_plugin(name, src = None, config = {}, **kwargs):
     if not types.is_dict(config):
         fail("config must be a dict, not a " + type(config))
 
+    # For backward compat
```

Comment:
Won't rules_js 2.0 be a breaking change and we don't need this? Should we print a warning?

Reply:
Warning and then we can drop it on the next major? 👍 

I do think we should minimize breaking changes in rulesets when we are forcing so many updates all at once. The rules_js breaking changes are enough already.